api-docs/influxdb3/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml

```
@@ -137,6 +137,141 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
   /query:
+    get:
+      operationId: GetV1ExecuteQuery
```

```suggestion
      operationId: GetQueryV1
```
Follow the `post` ID pattern.